.changeset/khaki-horses-lay.md

```diff
@@ -0,0 +1,18 @@
+---
+'playroom': minor
+---
+
+Adds multi-cursor support.
+
+The keyboard shortcuts added in the previous version (swap/duplicate line up/down) now support multiple cursors being on screen.
+"Select next occurrence" and "add cursor up/down" have also been implemented.
+
+| Keybinding     | Action                  |
+| -------------- | ----------------------- |
+| Alt-Up         | Swap line up            |
```

Comment:
What do you think about making these shortcuts more semantic / nicer to read?
```suggestion
| <kbd>Alt</kbd>-<kbd>Up</kbd>         | Swap line up            |
```

<img width="362" alt="keyboard" src="https://user-images.githubusercontent.com/3297808/222990228-3fd8b801-85ca-4ddd-bfe4-39720262ceb5.png">

Reply:
Oh what I've never even heard of kbd